src/test/java/uk/gov/digital/ho/proving/income/JacksonTest.java

```diff
@@ -0,0 +1,53 @@
+package uk.gov.digital.ho.proving.income;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.Ignore;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+import uk.gov.digital.ho.proving.income.api.Applicant;
+import uk.gov.digital.ho.proving.income.api.FinancialStatusRequest;
+
+import java.io.IOException;
+import java.time.LocalDate;
+import java.time.Month;
+import java.util.Arrays;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.springframework.boot.test.context.SpringBootTest.WebEnvironment.RANDOM_PORT;
+
+/**
+ * I've decided to keep this test as it is useful for debugging Jackson marshaling / unmarshaling errors in other tests
+ */
```

Comment:
It's an odd test to have, was this something you created for debugging purposes in the first place?

Reply:
I ended up writing the same class twice (for debugging purposes) so I wanted a copy of it saving somewhere.  But yes it is odd, so I have deleted it and can always pull it out of the history if required.